website/src/config-reference.md

```diff
@@ -594,16 +594,19 @@ Miscellaneous parameters
 
 ### `phpVersion`
 
-**default**: `null` (current PHP version is used)
+**default**: `null`
+
+PHPStan can be configured to analyse the codebase as if it was written for a different PHP version than you're currently running.
 
-If you want to analyse a codebase as if it was written for a different PHP version than you're currently running, change the `phpVersion` parameter:
+It determines the PHP version it should use for the analysis in the following order of priority:
 
+1. The `phpVersion` parameter
 ```yaml
 parameters:
     phpVersion: 70400 # PHP 7.4
 ```
-
-PHPStan will automatically infer the `config.platform.php` version from the last `composer.json` file it can find, if not configured in the PHPStan configuration file.
+2. `config.platform.php` version from the last `composer.json` file it can find.
```

Comment:
what does it mean by the "last composer.json file it can find?"

i appreciate that was already in the docs before your change but that sentence is quite puzzling and could do with some additional context

Reply:
It looks like it's searching for `composer.json` in autoloader project paths, see https://github.com/phpstan/phpstan-src/blob/1.9.1/src/Php/PhpVersionFactoryFactory.php#L38. the autoloader project paths are determined in https://github.com/phpstan/phpstan-src/blob/1.9.1/bin/phpstan directly, using the current working dir and some other logic